tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore2IastTests.cs

```diff
@@ -95,6 +97,30 @@ public async Task TestIastWeakHashingRequestSampling()
             await TestWeakHashing(filename, agent);
         }
 
+        [SkippableTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        [Trait("RunOnWindows", "True")]
+        public async Task TestIastSqlInjectionRequest(bool enableIast)
```

Comment:
Just a heads up, @zacharycmontoya has done a lot of refactoring in #3657 to remove the flakiness in the asm tests

Reply:
I will add the latest code from main and check if I need to update these new tests. Thanks!

Reply:
Yes, I had to update the new tests and make them like the others coming from master.  Done!